index.ts

```diff
@@ -1,30 +1,37 @@
 #!/usr/bin/env node
```

Comment:
```suggestion
#!/usr/bin/env tsx
```

node wont work anymore since this is typescript.

changing the things in the review then doing `npx .` works

Reply:
could leave this header as node if we're going to build the ts to js. Or we could replace `tsx` with `node` in the resulting js